File: Beginner/HOW MANY DIGITS DO I HAVE (HOWMANY)/digits.py
n = input()

d = 0

for i in range(len(n)):
    n = int(n)
    n = n/10
    d = d+1

if d==1:
    print(1)
elif d==2:
    print(2)
elif d==3:
    print(3)
else:
    print('More than 3 digits')

File: Beginner/HOW MANY DIGITS DO I HAVE (HOWMANY)/digits.cs
using System;

namespace Digits
{
    class Program
    {
        public static void Main(string[] args)
        {
            Console.WriteLine("Please enter a number: ");
            int n;
            string s = Console.ReadLine();
            
            if (int.TryParse(s, out n))
            {
                Console.WriteLine();
                switch (s.Length)
                {
                    case 1:
                        Console.WriteLine("1");
                        break;
                    case 2:
                        Console.WriteLine("2");
                        break;
                    case 3:
                        Console.WriteLine("3");
                        break;
                    default:
                        Console.WriteLine("More than 3 digits");
                        break;
                }
            }
            else
            {
                Console.WriteLine("You did not enter a number!");
            }
        }
    }
}
